runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -388,15 +387,18 @@ public void draw(MainBufferProvider mainBufferProvider, Graphics graphics, int x
 		final Image finalImage;
 		if (client.isStretchedEnabled())
 		{
-			GraphicsConfiguration gc = clientUi.getGraphicsConfiguration();
+			GraphicsConfiguration gc = ClientUI.getGraphicsDevice().getDefaultConfiguration();
 			Dimension stretchedDimensions = client.getStretchedDimensions();
 
 			if (lastStretchedDimensions == null || !lastStretchedDimensions.equals(stretchedDimensions)
-				|| (stretchedImage != null && stretchedImage.validate(gc) == VolatileImage.IMAGE_INCOMPATIBLE))
+				|| (stretchedImage != null && stretchedImage.validate(gc) != VolatileImage.IMAGE_OK))
```

Comment:
This is causing the volatile image to be completely reconstructed each frame on my system (Win 10)

Reply:
I determined that the first revalidate() call on a new image always returns RESTORED, which is what causes this to get in a rebuild loop.